app/views/companies/_companies_list.html.haml

```diff
@@ -33,8 +33,8 @@
                 <br/>
                 = bc.name
           %td= link_to company.name, company
-          %td= company.region&.name
-          %td= company.city
+          %td= company.addresses.first.region&.name
+          %td= company.addresses.first.city
```

Comment:
These statements will fail in production unless the rake task (included in this PR) is run first.  I assume this is the plan(?)